tests/security_and_privacy/test_private_session_awesome_bar_exclusion.py

```diff
@@ -0,0 +1,39 @@
+from selenium.webdriver import Firefox
+
+from modules.browser_object import Navigation
+from modules.browser_object_panel_ui import PanelUi
+
+YOUTUBE_URL = "https://www.youtube.com/"
+FACEBOOK_URL = "https://www.facebook.com/"
+AMAZON_URL = "https://www.amazon.com/"
+
+
+def test_websites_visited_in_private_browser_not_displayed_in_awesome_bar(
+    driver: Firefox,
+):
+    """
+    C101665 - Verify the visited websites from the Private Browsing session are not displayed inside the normal session
+    Awesome Bar
+    """
+
+    initial_window_handle = driver.current_window_handle
+
+    nav = Navigation(driver)
+    panel_ui = PanelUi(driver)
+    panel_ui.open_private_window()
+    panel_ui.switch_to_new_window()
+
+    for url in [YOUTUBE_URL, FACEBOOK_URL, AMAZON_URL]:
```

Comment:
stylistic thing, since you reuse this list, it might be easier to define it as a list at the top, it's up to you!